frontend/src/pages/modelServing/screens/projects/ModelServingPlatformSelectButton.tsx

```diff
@@ -47,8 +46,6 @@ const ModelServingPlatformSelectButton: React.FC<ModelServingPlatformSelectButto
           if (e instanceof Error) {
             setError(e);
           }
-        } finally {
-          setIsLoading(false);
```

Comment:
I think actually since we don't have the `finally` case anymore here we do need to call `setIsLoading(false)` along with `setError(e)` in the error case.